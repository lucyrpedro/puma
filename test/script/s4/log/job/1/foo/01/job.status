CYLC_BATCH_SYS_NAME=background
CYLC_BATCH_SYS_JOB_ID=5656
CYLC_BATCH_SYS_JOB_SUBMIT_TIME=2020-11-26T13:37:16Z
CYLC_JOB_PID=5656
CYLC_JOB_INIT_TIME=2020-11-26T13:37:16Z
CYLC_JOB_EXIT=SUCCEEDED
CYLC_JOB_EXIT_TIME=2020-11-26T13:37:17Z
