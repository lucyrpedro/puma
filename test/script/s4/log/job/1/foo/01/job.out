Suite    : s4
Task Job : 1/foo/01 (try 1)
User@Host: lucy@xfer1.jasmin.ac.uk

2020-11-26T13:37:16Z INFO - started
2020-11-26T13:37:17Z INFO - succeeded
